schema/synthetix/view_synths_rates.sql

```diff
@@ -0,0 +1,5 @@
+CREATE OR REPLACE VIEW synthetix.synths_rates AS
+SELECT currency_key_, currency_rate_, evt_block_time
+FROM synthetix."ExchangeRates_evt_RatesUpdated" r, unnest("currencyKeys", "newRates") as u(currency_key_, currency_rate_)
```

Comment:
This might not be correct https://www.postgresql.org/docs/current/queries-table-expressions.html#QUERIES-LATERAL